.circleci/config.yml

```diff
@@ -326,6 +327,23 @@ jobs:
           working_directory: ~/expo
           command: fastlane android prod_release
 
+  docs:
+    executor: nix
+    resource_class: xlarge # TODO: Use less than 8Gib of ram to build a static site
+    steps:
+      - setup
```

Comment:
Can you put this entire job behind an approval step? Because of the xlarge executor and the fact that most commits don't touch docs at all, I'd prefer to skip this job when it's not necessary. For now, human approval seems like the simplest and quickest way to build+deploy when needed.